var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -384,6 +379,15 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
 
     patch("kits_with_catalyst_5_12.patch", when="@5.12.0")
 
+    # Classic Intel compilers don't compile some code in vtkFFT.txx
+    # https://gitlab.kitware.com/vtk/vtk/-/merge_requests/11928
+    # Applying it for every compiler is fine.
+    patch("vtk-fft-icc.patch", when="@5.12.0:5.13.2")
+
+    # Patch for ParaView 5.13.0-5.13.2. Classic Intel compilers have a different
+    # keyword annotating attributes. Applying it for every compiler is fine.
+    patch("vtk-wrap-hierarchy.patch", when="@5.13.0:5.13.2")
```

Comment:
Is there any MR related to this change? If not we might want to keep the upper boundary of the version interval open

Reply:
They have been backported to VTK's `paraview/release` branch so any future 5.13.x versions will have them. Hence the upper boundary should stay at 5.13.2 as is IMO.

Reply:
Sounds good, Would you be able to add the link of MR/commit in the comment above for future references.

Reply:
Added.